src/ontology/uberon-edit.obo

```diff
@@ -224375,6 +224375,20 @@ relationship: part_of UBERON:0002395 ! talus
 property_value: http://purl.org/dc/elements/1.1/date 2021-05-28T01:09:52Z xsd:dateTime
 property_value: term:editor http://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500071
+name: rostrum
+def: "Rostrum is a common term used to describe forward protrusions from an animal's face. In vertebrates, this includes the part of the cranium (skull) in front of the zygomatic arches that also holds the teeth, palat, and nasal cavity. In invertebrates it is the forward extension of the carapace and/or the piercing mouthparts." []
```

Comment:
This isn't quite following best practice, we define the thing and don't talk about terminology in the definition

E.g.

"A rostrum is a forward protrusion in the face..." is better

Better still, no DRY:

"A forward protrusion..."

Better still use the genus

"An organism subdivision that is the protruding part of..."

(this can sometimes make for less biologist friendly terms but that itself suggests we need more intermediate terms or better labels for our abstract classes)

All this can be handled later we have a lot of definitions to fix

Reply:
@cmungall want me to revise the definition?